yb-voyager/src/tgtdb/event.go

```diff
@@ -183,6 +183,25 @@ func (e *Event) GetParams() []interface{} {
 	}
 }
 
+func (e *Event) GetParamsString() string {
+	params := e.GetParams()
+	paramsStr := ""
+	for _, p := range params {
+		pstr, ok := p.(*string)
+		if !ok {
+			// just as a safety check, this should never happen
+			paramsStr += "?, "
+			continue
+		}
+		if pstr != nil {
+			paramsStr += fmt.Sprintf("%s, ", *pstr)
+		} else {
+			paramsStr += "NULL, "
+		}
+	}
+	return paramsStr
+}
```

Comment:
I think it's unlikely that the perf impact, if any, will be significant. DB has usually been the bottleneck so far. 
That being said, since it's in the critical path, let me use https://pkg.go.dev/strings#Builder, that would be better.

Reply:
Here we are building and printing also each event so it will make it overall slower in case of debug.
StringBuilder seems a good choice

Reply:
Printing can make it slow, yes. But then again, if we are in debug mode, don't think we need to care about performance really :)

Reply:
agree to some extent. Just that we need to avoid it being dead slow.